src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -39,30 +39,34 @@ application:
 
 ```text
 my_flutter/
-├─.ios/
-│ ├─Runner.xcworkspace
-│ └─Flutter/podhelper.rb
-├─lib/
-│ └─main.dart
-├─test/
-└─pubspec.yaml
+├── .ios/
+│   ├── Runner.xcworkspace
+│   └─ Flutter/podhelper.rb
```

Comment:
Is is missing a `─` if you're adding one to each.

Reply:
🤦🏻‍♂️